libs/island-ui/core/src/lib/IconRC/icons/Accessibility.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react'
+interface SVGRProps {
```

Comment:
To make it easier to maintain if we need to change it, I would probably move this interface and the type out of each icon so you can have:

```
import { IconType } from '../Icon'

function SvgAccessibility({ title, titled, ...props }): IconType
```

The replace all feature will be very useful in this case 😅

Reply:
for consistency we could make a [custom template](https://react-svgr.com/docs/custom-templates/) for the generator and maybe keep the file in the Icon folder if the icon pack changes

Reply:
That would make sense! It would overwrite all the react components with the svg you are providing as entry files?

Reply:
I removed all the svg files, wasn't sure they belonged in the UI library. I could include them in the Icon folder for now and moving forward we can relocate them if we want.

Reply:
I think it's fine to not have them in the repo. I would just add the documentation on how to generate the react components and the template for svgr